metric/metric.go

```diff
@@ -38,9 +37,27 @@ type MeterProvider interface {
 //
 // An uninitialized Meter is a no-op implementation.
 type Meter struct {
-	impl MeterImpl
+	impl sdkapi.MeterImpl
 }
 
+// WrapMeterImpl constructs a `Meter` implementation from a
+// `MeterImpl` implementation.
+func WrapMeterImpl(impl sdkapi.MeterImpl) Meter {
+	return Meter{
+		impl: impl,
+	}
+}
+
+// Measurement is used for reporting a synchronous batch of metric
+// values. Instances of this type should be created by synchronous
+// instruments (e.g., Int64Counter.Measurement()).
+type Measurement = sdkapi.Measurement
+
+// Observation is used for reporting an asynchronous  batch of metric
+// values. Instances of this type should be created by asynchronous
+// instruments (e.g., Int64GaugeObserver.Observation()).
+type Observation = sdkapi.Observation
```

Comment:
Should these be deprecated in favor of the `sdkapi` package ones?

Reply:
I don't think so. I wrote about this in the PR description and the Changelog entry, but I've added a comment in the code for each of these to say that they are intentional aliases because they belong in both APIs.

I suppose we could replace these with two identical structs and copy them, I'm open to this if you think it's more appropriate.